dashboard/regen_mocks.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Copyright 2019 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+dart pub get
```

Comment:
```suggestion
flutter pub get
```

Reply:
I was getting a deprecation warning, but it probably came from the "flutter pub run" line below (now dart run).